traffic_ops/traffic_ops_golang/topology/topologies.go

```diff
@@ -600,6 +600,13 @@ func (topology *TOTopology) Update(h http.Header) (error, error, int) {
 	if len(topologies) != 1 {
 		return fmt.Errorf("cannot find exactly 1 topology with the query string provided"), nil, http.StatusBadRequest
 	}
+
+	// check if the name field was being updated by someone else
```

Comment:
This comment is misleading. This function just checks whether or not this entity (queried by its name) was modified or not. The modified field could be anything though. For ex, in case of topologies, the field could either be name or description.

Reply:
Same as below.